bigbluebutton-html5/imports/api/users-settings/server/methods/addUserSettings.js

```diff
@@ -24,6 +24,7 @@ const oldParameters = {
   presenterTools: 'bbb_presenter_tools',
   shortcuts: 'bbb_shortcuts',
   skipCheck: 'bbb_skip_check_audio',
+  skipCheckOnJoin: 'bbb_skip_check_audio_on_first_join',
```

Comment:
```suggestion
```

We are adding a new parameter, no need to support an old format of it - `skipCheckOnJoin` has not been ever supported afaik